lib/client.ts

```diff
@@ -147,7 +149,7 @@ class Client {
         })
     }
 
-    async send(requestCommand: string, requestData: any, destinationTwinId: number, expirationMinutes: number) {
+    async send(requestCommand: string, requestData: any, destinationTwinId: number, expirationMinutes: number, retries?: number) {
```

Comment:
I think we can set the default retries here on the method args
```suggestion
    async send(requestCommand: string, requestData: any, destinationTwinId: number, expirationMinutes: number, retries: number = this.retries) {
```